picard/coverart/providers/caa.py

```diff
@@ -107,8 +247,8 @@ def _has_suitable_artwork(self):
 
     def enabled(self):
         """Check if CAA artwork has to be downloaded"""
-        if not config.setting['ca_provider_use_caa']:
-            log.debug("Cover Art Archive disabled by user")
+        enabled = CoverArtProvider.enabled(self)
+        if not enabled:
             return False
```

Comment:
Why not `return not CoverArtProvider.enabled(self)` if we're not doing any logging here?